appveyor.yml

```diff
@@ -0,0 +1,19 @@
+version: 1.5.{build}
+image: Visual Studio 2017
+pull_requests:
+  do_not_increment_build_number: true
+branches:
+  only:
+  - master
+configuration: Release
+before_build:
+- cmd: dotnet restore
+build_script:
+- cmd: build.cmd Release
+after_test:
+- choco install opencover.portable
```

Comment:
Sounds like this will also add code coverage information to the project.

Super cool!!

Reply:
Yes